securedrop/journalist_app/forms.py

```diff
@@ -28,7 +28,7 @@ def minimum_length_validation(form: FlaskForm, field: Field) -> None:
     if len(field.data) < Journalist.MIN_USERNAME_LEN:
         raise ValidationError(
             ngettext(
-                'Must be at least {num} characters long.',
+                'Must be at least {num} character long.',
```

Comment:
Could also get rid of `{num}` here and say `1` or `one`, not sure if any option is better from the standpoint of translators

Reply:
I don't know. The only benefit I saw of replacing the placeholder was that `one` meets AP style rules, but we still fall afoul of them at `2`. :wink: 

Either option right now can cause a Weblate check complaint, so no benefit there. I would just go with what you have.